build.gradle.kts

```
@@ -32,7 +32,7 @@ allOpen {
 }
 
 group = "com.ampnet"
-version = "0.13.0"
+version = "0.14.0"
```

```suggestion
 version = "0.13.1"
```

Because frontend still didn't change the API for deposit.